4-appfactory/apps/cymbal-bank/main.tf

```diff
@@ -29,8 +29,10 @@ locals {
   expanded_app_services = flatten([
     for key, services in local.app_services : [
       for service in services : {
-        app_name     = key
-        service_name = service
+        app_name            = key
+        app_acronym         = "cb"
+        service_name        = service
+        create_env_projects = service == "userservice" || service == "ledgerwriter"
```

Comment:
Do we want `frontend` as well?

Reply:
It's also suggest using `contains()` as more compact, especially with a longer list which can be made a local.

Reply:
Otherwise this change LGTM :)

Reply:
I've just found usage for the app project when deploying alloydb

Reply:
SG - Thanks!